examples/react-vanilla-ts-src/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "example-react-vanilla-ts-src",
+  "type": "module",
+  "scripts": {
+    "dev": "tsx src/server.ts --dev",
+    "start": "NODE_ENV=production node build/server.js",
+    "build": "pnpm build:client && pnpm build:server",
+    "build:client": "vite build --app --emptyOutDir",
+    "build:server": "tsc && tsdown -d ./build src/server.ts",
```

Comment:
Why run both `tsc` AND `tsdown`? Isn't `tsdown` by itself enough?

Reply:
`tsdown` has no type checking, just transforming.

Reply:
The call to `tsc` must be changed to specify type checking only.

Reply:
Ah, I see. Then there should be no `outDir` in `tsconfig.json`, right?